### ISSUE TITLE
.v-step z-index

### ISSUE THREAD

**Issue by zoushenglin** [pulsardev/vue-tour#57]
Sometimes, it's blocked by higher levels of parent elements.

I've changed " .v-step's z-index to 999" in Source code

**Comment by mmorainville** [comment]
We could indeed rise it by default but there could always be other elements above it if, for example, somewhere in the page there is an element with a v-step > 999.
So as it's quite project-dependent we won't fix it, you juste have to override it as you did.
